src/NServiceBus.RabbitMQ/Administration/QueueCreator.cs

```diff
@@ -21,14 +21,23 @@ public Task CreateQueueIfNecessary(QueueBindings queueBindings, string identity)
             using (var connection = connectionFactory.CreateAdministrationConnection())
             using (var channel = connection.CreateModel())
             {
-                foreach (var receivingAddress in queueBindings.ReceivingAddresses)
+                var queueDeclaringTopology = routingTopology as IDeclareQueues;
+                if (queueDeclaringTopology != null)
                 {
-                    CreateQueueIfNecessary(channel, receivingAddress);
+                    queueDeclaringTopology.DeclareAndInitialize(
```

Comment:
What's the reason for the wrapping to a second line here and all the other places the method is used?

I personally find the code harder to parse with breaks in methods like this, and try to avoid it unless there is complex object/collection initialization syntax involved.